### PR TITLE
added hello.html

### PR DESCRIPTION
**Pull request by wminaung**
added hello.html please merge

### CODE REVIEW

File: hello.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Hello</title>
```
Comment (MayPhooPyaeSan) [inline hello.html:7]: please be updated.

Comment (MayPhooPyaeSan) [inline hello.html:7]: again please.

Comment (wminaung) [reply hello.html:7]: ok

Comment (wminaung) [reply hello.html:7]: ok

**Pull request merged**